packages/contracts-bedrock/scripts/DeployConfig.s.sol

```diff
@@ -33,8 +33,11 @@ contract DeployConfig is Script {
     uint256 public finalizationPeriodSeconds;
     address public proxyAdminOwner;
     address public baseFeeVaultRecipient;
+    uint256 public baseFeeVaultMinimumWithdrawalAmount;
     address public l1FeeVaultRecipient;
+    uint256 public l1FeeVaultMinimumWithdrawalAmount;
     address public sequencerFeeVaultRecipient;
+    uint256 public sequencerFeeVaultMinimumWithdrawalAmount;
```

Comment:
Why did we add new state vars here? I see they were already in the config file and just not read in previously. Presuambly this means they were used in tests but not in the actual deploy process?

Reply:
These were previously not used in the solidity code but now are due to us setting up the L2 predeploys using the deploy config. Previously this was done in the Go code that generates the L2 genesis state